### PR TITLE
Bottom Padding on Project Card when no Published At Time

### PR DESCRIPTION
**Pull request by dmcavoy**
Adding more padding at bottom of project card when time is hidden.
screen shot 2019-02-01 at 6 55 58 pm

### CODE REVIEW

File: apps/src/templates/projects/ProjectCard.jsx [hunk 1]
```diff
@@ -105,6 +108,11 @@ export default class ProjectCard extends React.Component {
       Object.assign(thumbnailStyle, styles.fullThumbnail);
     }
 
+    const noTimeOnCardStyle = {};
```
Comment (maddiedierker) [inline apps/src/templates/projects/ProjectCard.jsx]: lines 111-113 could simplify to:

```
const cardStyle = this.props.isDetailView ? {} : styles.noTime;
```

which makes the `<div style={cardStyle}>` more explicit without needing extra context

**Review by maddiedierker — APPROVED** [review]
left a nit comment, but this looks great! thanks for the quick fix 😄

**Review by Hamms — COMMENTED** [review]
Do we want this padding for the scenario where we show the `updatedAt` timestamp?

It also looks like this doesn't exactly cover the case when we show the `publishedAt`, since we also consider `isPublicGallery` and `projectData.publishedAt` for the actual view logic. We should probably pull out `isPublicGallery && isDetailView && projectData.publishedAt` into a `shouldShowPublishedAt()` helper, and use that for both.

**Comment by dmcavoy** [comment]
@Hamms You are right - it was not specific enough. I think I have updated now so that it solves for your points.

**Comment by Hamms** [comment]
Looks like we're still missing the `updatedAt` case, unless I'm misunderstanding something

**Comment by dmcavoy** [comment]
@Hamms updatedAt only applies to the Personal Gallery and the hiding of the time published on the card is only for the public gallery

**Comment by Hamms** [comment]
Right, but we probably don't want the padding in the scenario where `updatedAt` is shown

**Comment by Hamms** [comment]
I'm expecting something like 

```js
const cardStyle = (shouldShowPublishedAt || shouldShowUpdatedAt) ? {} : styles.noTime;
```

**Comment by dmcavoy** [comment]
Oh I see what you are saying. I was only solving for the new case we added for the public gallery. I'm pretty sure because the age and name are just for the public gallery that this padding add does not impact it. What I'm now wondering is how do you see the personal gallery view anyway? Is this what shows up on your homepage?

**Comment by Hamms** [comment]
Ahh, gotcha. Didn't realize this was influenced by the name and age!

And now that you mention it, I'm not sure! Can you add a jira item to confirm whether or not the personal gallery view option for this can be removed?

**Review by Hamms — APPROVED** [review]

**Comment by dmcavoy** [comment]
I think I just found the personal gallery view - i think its on the homepage

**Pull request merged**